pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/creator/impl/SegmentIndexCreationDriverImpl.java

```diff
@@ -272,6 +272,9 @@ private void handlePostCreation()
       // Write the index files to disk
       indexCreator.setSegmentName(segmentName);
       indexCreator.seal();
+    } catch (Exception e) {
+      LOGGER.error("Failed to seal indexCreator", e);
```

Comment:
Ideally we should either log error or throw the exception, or we might log the same exception twice